src/movies.js

```diff
@@ -1,22 +1,94 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+   const newMovieArr = moviesArray.map(function(element){
+        return element.director
+    })
+
+//    const filteredArr = newMovieArr.filter(function(director, index, array) {
+//     return array.indexOf(director) === index;
+// })
+// i checked with AI but dont completely understand the parameters and how is automatically plugged what it needs
```

Comment:
were you can do something like this, this cleans the array 

`  return [...new Set(newMovieArr)]`